src/tools/wasm-merge.cpp

```diff
@@ -235,6 +235,19 @@ void updateNames(Module& wasm, KindNameUpdates& kindNameUpdates) {
     }
 
   private:
+    Name& resolveName(NameUpdates updates, Name& name) {
```

Comment:
```suggestion
    Name& resolveName(NameUpdates& updates, Name& name) {
```
Without this change we copy that map each time, which can be slow.